crates/accelerate/src/convert_2q_block_matrix.rs

```diff
@@ -32,32 +37,75 @@ pub fn blocks_to_matrix(
     py: Python,
     op_list: Vec<(PyReadonlyArray2<Complex64>, SmallVec<[u8; 2]>)>,
 ) -> PyResult<Py<PyArray2<Complex64>>> {
-    let identity = aview2(&ONE_QUBIT_IDENTITY);
-    let input_matrix = op_list[0].0.as_array();
-    let mut matrix: Array2<Complex64> = match op_list[0].1.as_slice() {
-        [0] => kron(&identity, &input_matrix),
-        [1] => kron(&input_matrix, &identity),
+    let identity = aview2(&ONE_QUBIT_IDENTITY).into_faer_complex();
+    let input_matrix = op_list[0].0.as_array().into_faer_complex();
+
+    let mut matrix = match op_list[0].1.as_slice() {
+        [0] => identity.kron(input_matrix.as_ref()),
+        [1] => input_matrix.kron(identity.as_ref()),
```

Comment:
Is `as_ref()` needed here? Aren't these both `MatRef` already?
```suggestion
        [0] => identity.kron(input_matrix),
        [1] => input_matrix.kron(identity),
```

Reply:
Oh, that's right, thanks! Removed in the last commit :)